debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (23.1.1-0ubuntu2) UNRELEASED; urgency=medium
```

Comment:
I updated the version number here from 23.2.... to 23.1.1.... ubuntu2. it'll get rewritten when we actually release anyway so no big deal that it was 23.2